<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="3" ignored="0" total="5" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="39200" started-at="2019-02-07T05:35:18Z" finished-at="2019-02-07T05:35:58Z">
    <groups>
      <group name="Login">
        <method signature="LoginPageTest.validLoginTest()[pri:1, instance:com.seleniumassignment.qa.testcases.LoginPageTest@5891e32e]" name="validLoginTest" class="com.seleniumassignment.qa.testcases.LoginPageTest"/>
      </group> <!-- Login -->
    </groups>
    <test name="Default test" duration-ms="39200" started-at="2019-02-07T05:35:18Z" finished-at="2019-02-07T05:35:58Z">
      <class name="com.seleniumassignment.qa.testcases.LoginPageTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.seleniumassignment.qa.testcases.LoginPageTest@5891e32e]" name="setUp" is-config="true" duration-ms="7724" started-at="2019-02-07T05:35:18Z" finished-at="2019-02-07T05:35:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="LogoTest()[pri:0, instance:com.seleniumassignment.qa.testcases.LoginPageTest@5891e32e]" name="LogoTest" duration-ms="62" started-at="2019-02-07T05:35:26Z" description="Verify Logo image" finished-at="2019-02-07T05:35:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LogoTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.seleniumassignment.qa.testcases.LoginPageTest@5891e32e]" name="tearDown" is-config="true" duration-ms="937" started-at="2019-02-07T05:35:26Z" finished-at="2019-02-07T05:35:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.seleniumassignment.qa.testcases.LoginPageTest@5891e32e]" name="setUp" is-config="true" duration-ms="6054" started-at="2019-02-07T05:35:27Z" finished-at="2019-02-07T05:35:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="invalidLoginTest()[pri:0, instance:com.seleniumassignment.qa.testcases.LoginPageTest@5891e32e]" name="invalidLoginTest" duration-ms="987" started-at="2019-02-07T05:35:33Z" description="Verify invalid login" finished-at="2019-02-07T05:35:34Z">
          <exception class="java.lang.AbstractMethodError">
            <message>
              <![CDATA[com.seleniumassignment.qa.utils.WebEventListener.beforeGetText(Lorg/openqa/selenium/WebElement;Lorg/openqa/selenium/WebDriver;)V]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AbstractMethodError: com.seleniumassignment.qa.utils.WebEventListener.beforeGetText(Lorg/openqa/selenium/WebElement;Lorg/openqa/selenium/WebDriver;)V
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.openqa.selenium.support.events.EventFiringWebDriver.lambda$new$0(EventFiringWebDriver.java:84)
at com.sun.proxy.$Proxy6.beforeGetText(Unknown Source)
at org.openqa.selenium.support.events.EventFiringWebDriver$EventFiringWebElement.getText(EventFiringWebDriver.java:462)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
at com.sun.proxy.$Proxy11.getText(Unknown Source)
at com.seleniumassignment.qa.pages.LoginPage.invalidLogin(LoginPage.java:72)
at com.seleniumassignment.qa.testcases.LoginPageTest.invalidLoginTest(LoginPageTest.java:48)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AbstractMethodError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidLoginTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.seleniumassignment.qa.testcases.LoginPageTest@5891e32e]" name="tearDown" is-config="true" duration-ms="866" started-at="2019-02-07T05:35:34Z" finished-at="2019-02-07T05:35:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.seleniumassignment.qa.testcases.LoginPageTest@5891e32e]" name="setUp" is-config="true" duration-ms="5874" started-at="2019-02-07T05:35:35Z" finished-at="2019-02-07T05:35:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="tabsTest()[pri:0, instance:com.seleniumassignment.qa.testcases.LoginPageTest@5891e32e]" name="tabsTest" duration-ms="0" started-at="2019-02-07T05:35:41Z" description="Verify 4 tabs" finished-at="2019-02-07T05:35:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tabsTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.seleniumassignment.qa.testcases.LoginPageTest@5891e32e]" name="tearDown" is-config="true" duration-ms="859" started-at="2019-02-07T05:35:41Z" finished-at="2019-02-07T05:35:42Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.seleniumassignment.qa.testcases.LoginPageTest@5891e32e]" name="setUp" is-config="true" duration-ms="6061" started-at="2019-02-07T05:35:42Z" finished-at="2019-02-07T05:35:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="validLoginTest()[pri:1, instance:com.seleniumassignment.qa.testcases.LoginPageTest@5891e32e]" name="validLoginTest" duration-ms="954" started-at="2019-02-07T05:35:48Z" description="Valid Login" finished-at="2019-02-07T05:35:49Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Altoro Mutual: Online Banking Home] but found [Altoro Mutual]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Altoro Mutual: Online Banking Home] but found [Altoro Mutual]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.seleniumassignment.qa.testcases.LoginPageTest.validLoginTest(LoginPageTest.java:62)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validLoginTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.seleniumassignment.qa.testcases.LoginPageTest@5891e32e]" name="tearDown" is-config="true" duration-ms="984" started-at="2019-02-07T05:35:49Z" finished-at="2019-02-07T05:35:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.seleniumassignment.qa.testcases.LoginPageTest@5891e32e]" name="setUp" is-config="true" duration-ms="6831" started-at="2019-02-07T05:35:50Z" finished-at="2019-02-07T05:35:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="loginPageTtileTest()[pri:2, instance:com.seleniumassignment.qa.testcases.LoginPageTest@5891e32e]" name="loginPageTtileTest" duration-ms="5" started-at="2019-02-07T05:35:57Z" description="Verify pageTitle" finished-at="2019-02-07T05:35:57Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Altoro Mutual: Online Banking Login] but found [Altoro Mutual]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Altoro Mutual: Online Banking Login] but found [Altoro Mutual]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.seleniumassignment.qa.testcases.LoginPageTest.loginPageTtileTest(LoginPageTest.java:43)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginPageTtileTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.seleniumassignment.qa.testcases.LoginPageTest@5891e32e]" name="tearDown" is-config="true" duration-ms="967" started-at="2019-02-07T05:35:57Z" finished-at="2019-02-07T05:35:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.seleniumassignment.qa.testcases.LoginPageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
